.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -8,7 +8,7 @@ about: Suggest an idea for this project
 <!-- A clear and concise description of what the problem is. -->
 
 **Describe the solution you'd like**
-<!-- A clear and concise description of what you want to happen. -->
+<!-- A clear and concise description of what you want to happen. Provide examples of use cases that this would enable and how it solves the problem described above. -->
```

Comment:
I'd prefer explicitly saying "*real world* use cases". The key here is to make it clear to people that we are interested in actual examples, not theoretical possibilities.

Reply:
Otherwise, LGTM.